.circleci/config.yml

```diff
@@ -22,12 +22,15 @@ jobs:
       name: default
     steps:
       - checkout
+      - attach_workspace:
+          at: ~/go/mattermost/
       - run:
+          name: "Validate lint"
           command: |
+            echo "installing golangci-lint"
+            curl -sfl https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.21.0 # keep this in sync with .golangci.yml
```

Comment:
The Plugin Marketplace also uses golangci with circleci. You might want to just copy and past the command from there: https://github.com/mattermost/mattermost-marketplace/pull/27/files#diff-1d37e48f9ceff6d8030570cd36286a61

The advantage of this implementation is, that the binary gets cached, which speeds up the build.

Reply:
Oh good point. I've copied from our service circleci file, but this one is better. Thank you